### PR TITLE
Bump version 0.7

### PR DESCRIPTION
**Pull request by hechi**
TL;DR
Release miflora to version 0.7 which includes [btlewrap==0.0.9](https://github.com/open-homeautomation/miflora/commit/ebda66d1f4ba71bc0b98f8383280e59302b40fc8)
---
Hey,
first thanks for the nice work you did on this project, it's super awesome. I recently bought some miflora devices and now I am looking forward to using them :) but sadly there is a dependency on the btlewrap and it would be nice to have version 0.0.9 included. I saw that's already upgraded but not released yet. 
Would be awesome if you could release it :) 

I bumped the version in the `setup.py` and executed the `./build.sh` script. 

Thx

### CODE REVIEW

**Comment by ThomDietrich** [comment]
Everything checks out 👍

**Comment by ChristianKuehnel** [comment]
I do not see any build results, it looks like the TravisCI integration is broken.

@open-homeautomation , could you please check the CI?

**Comment by hechi** [comment]
Hey @ChristianKuehnel and @ThomDietrich anything i can help with?

**Comment by ChristianKuehnel** [comment]
Sorry, right now @open-homeautomation is the only one who can create releases...

**Comment by hifiberry** [comment]
No time for this at the moment, if somebody wants to do this, let me know your PyPi user name.

**Comment by open-homeautomation** [comment]
No time to do this right now. If somebody wants to do this, let me know your PyPi username

**Comment by basnijholt** [comment]
I would be able to do the releases. Otherwise, I'm planning on forking and releasing under a different name.

**Comment by open-homeautomation** [comment]
I can just hand this repository over. For PyPi I need to know your PyPi username to transfer it.

**Comment by basnijholt** [comment]
@open-homeautomation, that's great. My PyPI username is `basnijholt`.

**Comment by open-homeautomation** [comment]
Ok, you should now have full access to both Github and the PyPi release. Please check if everything works.

**Comment by basnijholt** [comment]
Thanks a lot! I tried merging https://github.com/open-homeautomation/miflora/pull/150 but have no access yet.

**Comment by open-homeautomation** [comment]
You first have to accept the invite. Check your emails for a Github mail.

**Comment by basnijholt** [comment]
That's right, my bad! I will first setup GitHub actions to successfully run the tests and then release a new version :)

**Comment by basnijholt** [comment]
I have just released v0.7.0 :tada:

https://pypi.org/project/miflora/#files
